tests/test_did.py

```diff
@@ -355,6 +361,7 @@ def test_fully_interacted(unit, cluster):
     )
 
 
+@pytest.mark.against_r_core
```

Comment:
This might fail in the CI as I don't think the data is available on github? It's from the did R repo: https://github.com/bcallaway11/did/tree/master/data When I wrote the test, I did not want to think about how to load dta files into pandas, and I did not simply want to copy over the file as I wasn't sure about the license. `did` uses [GPL-2](https://github.com/bcallaway11/did/blob/32a1da2427aa986c45c2b714847b8464d64ed72b/DESCRIPTION#L9C9-L9C15) which I think (?) does not allow reuse under any other license then GPL-2.